fdbbackup/tests/s3_backup_test.sh

```diff
@@ -82,10 +82,14 @@ function restore {
   local local_scratch_dir="${2}"
   local local_url="${3}"
   local local_credentials="${4}"
+
+  # enable read cache randomly to test both code paths.
+  local erc_value=$((RANDOM % 2))
```

Comment:
Instead of setting here, can we set the knob in ClientKnobs.cpp so that it covers the future S3 simulation changes too?